HW03_Julia.md

```diff
@@ -0,0 +1,197 @@
+
+``` r
+library("ggplot2")
+library(foreign) #needed to import my dataset
+library(scales) #needed to later convert my x-axis labels from decimal to percent
+```
+
+    ## 
+    ## Attaching package: 'scales'
+
+    ## The following object is masked from 'package:purrr':
+    ## 
+    ##     discard
+
+    ## The following object is masked from 'package:readr':
+    ## 
+    ##     col_factor
+
+**Importing my dataset as an R data.frame:**
+
+``` r
+Covid_data <- read.spss("Pathways April 2020 (ATP W66).sav", to.data.frame = TRUE) 
+```
+
+**The survey question I will be examining is**: “All in all, do you
+think that the coronavirus outbreak has been…”
+
+  - made a bigger deal than it really is
+
+  - made a smaller deal than it really is
+
+  - approached about right
+
+  - no answer
+
+A lot of my time was spent trying to understand how this dataset was
+organized. I found this
+[document](https://www.pewresearch.org/wp-content/uploads/2018/05/Codebook-and-instructions-for-working-with-ATP-data.pdf)
+that explained what the demographic profile variables meant. The
+variables corresponding to each survey question response were included
+in this document titled [Pathways April 2020 ATP
+W66](Pathways_April%202020_ATPW66_%20Topline.pdf) which comes with the
+dataset download. I am interested in responses as stratified by age
+(F\_AGECAT) and self-reported political party (F\_PARTY\_FINAL).
+
+``` r
+#names(Covid_data) #variables corresponding to each survey question
+```
+
+I will be plotting weighted survey estimates based on the variable
+WEIGHT\_W66 because “data for all Pew Research Center reports are
+analyzed using this weight.”
+
+First, I am cleaning up my dataframe and limiting the dataset to only
+the columns I need.
+
+``` r
+#limit to select columns, also renaming some columns
+select_Covid_data <- Covid_data %>%
+  select(weight = WEIGHT_W66, response = COVIDDEAL_W66, Age = F_AGECAT, Party = F_PARTY_FINAL) 
+
+head(select_Covid_data) #checking that it worked
+```
+
+    ##      weight                             response   Age      Party
+    ## 1 0.2948638               Approached about right   65+   Democrat
+    ## 2 0.6897131 Made a BIGGER deal than it really is   65+ Republican
+    ## 3 0.4420582               Approached about right   65+ Republican
+    ## 4 0.4231892               Approached about right 30-49   Democrat
+    ## 5 0.3680737               Approached about right 50-64   Democrat
+    ## 6 0.5717572               Approached about right 50-64 Republican
+
+I am rearranging the data so that I can easily calculate weighted
+summary stats by each demographic group by reshaping data to a “long”
+format. I’m not sure how to describe this, but [this
+article](https://medium.com/pew-research-center-decoded/using-tidyverse-tools-with-pew-research-center-survey-data-in-r-bdfe61de0909)
+touches upon it; this article was also just super helpful in general for
+formatting Pew Research data sets.
```

Comment:
I took a look at that article - it is definitely interesting and I will keep the long format in mind for the future!